GameFiles/GameInterface/_integration_iw4x.gsc

```diff
@@ -199,6 +199,11 @@ Log2Console( logLevel, message )
     PrintConsole( "[" + logLevel + "] " + message + "\n" );
 }
 
+_GetXUID()
```

Comment:
No this is wrong. See how `ISTestClient` function is called in the same file.
You need to use the `call` keyword.